internal/git/git_test.go

```diff
@@ -80,6 +80,45 @@ func TestCredentials(t *testing.T) {
 	}
 }
 
+func TestIsKnownGitError(t *testing.T) {
```

Comment:
Any reason for moving this test around?

Reply:
No 🙄  It was all moved elsewhere. And the some moved back, so that's why.